scripts/bump-version.js

```diff
@@ -1,21 +1,47 @@
 #!/usr/bin/env node
 /* eslint-env node */
 
-const sh = require('shelljs')
-const path = require('path')
-
-sh.set('-e')
+const sh = require("shelljs")
+const path = require("path")
 
+sh.set("-e")
 const lernaConfigPath = path.join(__dirname, '..', 'lerna.json')
 const rootPkgPath = path.join(__dirname, '..', 'package.json')
+const rootPkgLockPath = path.join(__dirname, '..', 'package-lock.json')
 
 const main = () => {
     sh.exec(`lerna version --no-push --no-git-tag-version --yes ${process.argv.slice(2).join(' ')}`)
     sh.exec(`npm install`)
     const lernaConfig = JSON.parse(sh.cat(lernaConfigPath))
     const rootPkg = JSON.parse(sh.cat(rootPkgPath))
+    const rootLockPkg = JSON.parse(sh.cat(rootPkgLockPath))
+
+    // find all monorepo packages, look inside each package json, find peerDependency that is a monorepo package
+    // and update it with a new version
+    sh.exec('npm run --silent lerna -- list --all --json', {silent: true}, function(code, stdout) {
+        const packages = JSON.parse(stdout.toString())
+
+        const lernaPackageNames = packages.map(pkg => pkg.name)
+        packages.forEach(async ({location}) => {
```

Comment:
```suggestion
        const promises = packages.map(async ({location}) => {
```
Using an async function inside of `forEach` means that any errors thrown in the function are silently ignored. Using `.map` instead means you can then use `Promise.all` or `Promise.allSettled` to handle any errors.

Reply:
Sorry, my bad, no need to use async here anymore. I was using it for `fs.readFile` originally, but I switched back to sh.cat